public/keymaps/nafuda_default .json

```
@@ -0,0 +1,28 @@
+{
+  "keyboard": "naked60/rev1",
+  "keymap": "default_ad36bb3",
+  "commit": "687a24f2980e6e490ac8e43104f4176b033c9243",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+                          "TG(1)",
+"LT(3, KC_BSPC)",         "KC_UP",     "TG(2)",
```

`LT()` keycodes don't import correctly if there are spaces in the value.

```suggestion
 "LT(3,KC_BSPC)",         "KC_UP",     "TG(2)",
```